### DISCUSSION TITLE
My farmed unspent XCH is not showing in wallet but showing in ChiaExplorer

### DISCUSSION THREAD

**Discussion by Adamadas** [Chia-Network/chia-blockchain#8338] [Support]
When serching my address in chiaexplorer    xch1fthy33zupt4ja2czr4s24uf9zcmxcrfjhe7udfrup3jfd7rtkheqvxwm7r it shows balance of 2 XCH and it shows 8 XCH Unspent ... My GUI wallet shows  only 10 XCH  farmed and 2  XCH available. Shouldn't  my lalance on this account be 8 XCH? This all happened while farmming solo. Now I am farming in pool to another account in the same wallet.
How can regain my lost XCH  in my wallet? I have reistalled the GUI Wallet  on Ubuntu on virgin (to Chia blockchain)  server and  on laptop - both  comps show  (after syncing) the same balance. 
Please help to solve the issue.

**Comment by BrandtH22** [comment]
@Adamadas , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
